rotordc_sale_picking_state/models/sale_order.py

```diff
@@ -0,0 +1,74 @@
+# SPDX-FileCopyrightText: 2022 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+from odoo import api, fields, models
+from odoo.tools.translate import _
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    @api.model
+    def get_picking_state_selection(self):
+        return [
+            # No stock picking found.
+            ("none", _("None")),
+            # Multiple conflicting found.
+            ("unknown", _("Unknown")),
+            # Same as states on stock.picking.
+            # FIXME: Should we dynamically create these from stock.picking? What
```

Comment:
Maybe you can indeed use the existing fields. For instance with `self.env["stock.picking"]._fields["state"].selection` 

Based on this [post](https://www.odoo.com/forum/help-1/how-to-generate-translation-terms-for-field-values-translate-true-did-not-work-184809), you can just set the field as translatable with `fields.Selection(..., translate=True)`. so there would be no need to use the "_".

Reply:
done

Reply:
Although I set `translate=True`, the selections do not become translatable. Reverting this change.

Reply:
Ah ok, maybe it was a bad idea altogether. I am not sure it is worth keeping the `state` options if they are not translatable...